atlp-rwanda/hackers-ec-be
# #187419060 User should be able to update their password

## What does this PR do?

User should be able to update their password

##Description of Task to be completed?

A user can update their password

## How should this be manually tested?

- Chrone Repository into your local machine
- Run npm install
- Fill all environment variables in .env file as they appear in .env example file
- Run npm run dev
- Use Postman to create a user on the endpoint: api/v1/users/register
- Use Postman to update the password on the endpoint: api/v1/users//password-update

## What are the relevant pivotal tracker/Trello stories?

187419060
